docs/getting-started/integrations/vuejs-v3.md

```diff
@@ -88,49 +105,54 @@ export default {
 			editor: ClassicEditor,
 			editorData: '<p>Hello from CKEditor 5 in Vue!</p>',
 			editorConfig: {
-			  plugins: [ Bold, Essentials, Italic, Mention, Paragraph, SlashCommand, Undo ],
-			  toolbar: [ 'undo', 'redo', '|', 'bold', 'italic' ],
-			  licenseKey: '<YOUR_LICENSE_KEY>',
-			  mention: { 
-				  // Mention configuration
-			  },
-			},
+				plugins: [ Bold, Essentials, Italic, Mention, Paragraph, SlashCommand, Undo ],
+				toolbar: [ 'undo', 'redo', '|', 'bold', 'italic' ],
+				licenseKey: '<YOUR_LICENSE_KEY>',
+				// Other configuration options...
+			}
 		};
 	}
 }
 </script>
 ```
 
-### Using the component locally
+#### Using the `<ckeditor>` component globally
```

Comment:
This one is kinda unclear, the subheading says globally, but the very first sentence explains local usage. Also, the local usage has pretty bad discoverability due to that.

Reply:
Oh, my bad. This one was suppose to say `locally`, not `globally`.